websites/T/Tsuki Mangás/dist/metadata.json

```diff
@@ -1,23 +1,17 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.6",
 	"author": {
-		"name": "fror",
-		"id": "515668127829458945"
+		"name": "Slowftw",
```

Comment:
Unless you have been given explicit approval from Slowftw, you cannot do this. Transfer the presence to the PreMiD bot account and leave Slowftw as a contributor

Reply:
I have talked with @Slowftw and he agreed to take over it.

Reply:
yo I agree 🤙